modules/sfp_trashpanda.py

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
+# -------------------------------------------------------------------------------
+# Name:         sfp_trashpanda
+# Purpose:      Spiderfoot plugin to query Trashpanda - got-hacked.wtf API to gather intelligence about
+#               mentions of your target in paste sites like Pastebin, Ghostbin and Zeropaste
+#
+# Author:      Krishnasis Mandal <krishnasis@hotmail.com>
+#
+# Created:     17/04/2021
+# Copyright:   (c) Steve Micallef
+# Licence:     GPL
+# -------------------------------------------------------------------------------
+
+import json
+import base64
+
+from spiderfoot import SpiderFootEvent, SpiderFootPlugin
+
+
+class sfp_trashpanda(SpiderFootPlugin):
+
+    meta = {
+        'name': "Trashpanda",
+        'summary': "Queries got-hacked.wtf to gather intelligence about metions of target in pastesites",
+        'flags': ["apikey"],
+        'useCases': ["Footprint", "Investigate", "Passive"],
+        'categories': ["Real World"],
+        'dataSource': {
+            'website': "https://got-hacked.wtf",
+            'model': "COMMERCIAL_ONLY",
+            'references': [
+                "http://api.got-hacked.wtf:5580/help"
+            ],
+            'favIcon': "https://seon.io/assets/favicons/favicon-16x16.png",
+            'logo': "https://got-hacked.wtf/wp-content/uploads/2020/07/cropped-IMG_7619.jpg",
+            'description': "The bot searches different paste sites for leaked credentials."
+            "The API itself gives access to all unique credentials the bot ever detected.",
+        }
```

Comment:
Instructions to obtain the API key are to follow the instructions at https://got-hacked.wtf/